code/WorkInProgress/laundry.dm

```diff
@@ -15,7 +15,9 @@
 	var/open = 0
 	var/cycle = PRE
 	var/cycle_time = 10
+	var/cycle_time_mob_inside = 5
```

Comment:
This (along with the other cycle time stuff) should be defines instead of vars if they aren't ever changed.
of course, they COULD be changed, with an EMAG or something..............

Reply:
Done. Tell me if i messed up.